saleor/thumbnail/views.py

```diff
@@ -52,7 +53,11 @@ def handle_thumbnail(request, instance_id: str, size: str, format: str = None):
     ).first():
         return HttpResponseRedirect(thumbnail.image.url)
 
-    instance = model_data.model.objects.get(id=pk)
+    try:
+        instance = model_data.model.objects.get(id=pk)
+    except ObjectDoesNotExist:
+        return HttpResponseNotFound("Cannot found instance with the given id.")
```

Comment:
I think it should be either :
"Cannot find instance with given id." 
or 
"Instance with given id was not found."

Reply:
or "Instance with given id cannot be found."